### DISCUSSION TITLE
login and register

### DISCUSSION THREAD

**Discussion by PriyanshiTyagi-07** [otahina/PowerPoint-Generator-Python-Project#10] [General]
there is a issue when i click on login and register
![Uploading Screenshot 2024-01-11 at 11.20.42 PM.png…]()

**Comment by otahina** [comment]
Hi @PriyanshiTyagi-07! Thank you for reporting. It looks like the screenshot you provided here is not correctly shown. Could you try to attach the image again? It would help me understand the situation.
